From d30fd13d3d40fc278ee58d9488a69cc4cdd67891 Mon Sep 17 00:00:00 2001
From: Jonathan Bell <jonathan@raspberrypi.com>
Date: Tue, 11 Feb 2025 14:15:41 +0000
Subject: [PATCH] dts: bcm2712: add missing brcm,vdm-qos-map to cm5 base dts

Without this, a default of 0 is used which is very suboptimal for timely
service. Consistency with Pi 5 is desired.

Signed-off-by: Jonathan Bell <jonathan@raspberrypi.com>
---
 arch/arm64/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

--- a/arch/arm64/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi
+++ b/arch/arm64/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi
@@ -118,6 +118,10 @@ rp1_target: &pcie2 {
 	status = "okay";
 };
 
+&pcie1 {
+	brcm,vdm-qos-map = <0x33333333>;
+};
+
 // Add some labels to 2712 device
 
 // The system UART
